kolibri/plugins/coach/api.py

```diff
@@ -212,12 +216,15 @@ def get_queryset(self):
         limit = self.check_limit()
         if before and limit:
```

Comment:
Likewise for before - I feel like it makes sense to change the whole semantics of the API endpoint to be focused on timestamps rather than ids.